xcube/core/store/descriptor.py

```diff
@@ -125,29 +126,27 @@ def __init__(self,
                  name: str,
                  dtype: str,
                  dims: Sequence[str],
-                 description: Optional[str] = '',
                  attrs: Mapping[str, any] = None):
         assert_given(name, 'name')
         assert_given(dtype, 'dtype')
         self.name = name
         self.dtype = dtype
         self.dims = tuple(dims)
         self.ndim = len(self.dims)
-        self.description = description
         self.attrs = dict(attrs) if attrs is not None else None
 
     @classmethod
     def from_dict(cls, d: Mapping[str, Any]):
         """Create new instance from a JSON-serializable dictionary"""
-        assert 'name' in d
-        assert 'dtype' in d
-        assert 'dims' in d
-        return VariableDescriptor(d['name'], d['dtype'], d['dims'], d.get('description', ''), d.get('attrs', None))
+        assert_in('name', d)
+        assert_in('dtype', d)
+        assert_in('dims', d)
```

Comment:
Das war vorher richtig(er): name, dtype, dims sind ja keine parameter, gehören also zum state vom descriptor. Wenn diese falsch sind, ist der constructor falsch aufgerufen worden damit geheoren die `assert_in` in den konstructor und hiert ist `assert` richtig(er).